src/test/java/uk/co/evoco/webdriver/configuration/driver/ConfiguredChromeDriverTests.java

```diff
@@ -3,11 +3,15 @@
 import org.apache.commons.io.FileUtils;
 import org.junit.jupiter.api.Test;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.support.events.EventFiringWebDriver;
 
 import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Map;
 
 import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 public class ConfiguredChromeDriverTests {
```

Comment:
What about an integration test where we make sure these settings are actually applied in the browser?  This test is fine for covering lines of code and ensuring that the configuration is making it into the map, but we're not testing that we're getting the desired behaviour in the actual browser.  Same comment for Firefox options I guess!  Please address.

Reply:
added a unit test for to test browser preferences